changelog.md

```diff
@@ -1,3 +1,34 @@
+ <a name="4.1.0"></a>
+# 4.1.0 (2019-01-28)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v4.0.0...v4.1.0)
+
+## Core
+
+* font-size audit: count code points for text length ([#6973](https://github.com/googlechrome/lighthouse/pull/6973))
```

Comment:
I think we had a new contributor for this one

https://github.com/GoogleChrome/lighthouse/commits?author=castilloandres

Reply:
> I think we had a new contributor for this one

good catch